spec/features/avatar_spec.rb

```diff
@@ -0,0 +1,19 @@
+require "rails_helper"
+
+feature "profile photo" do
+  scenario "user uploads a profile photo" do
+    visit "/users/sign_up"
+
+    fill_in "Email", with: "ash@s-mart.com"
+    fill_in "Password", with: "boomstick!3vilisd3ad"
+    fill_in "Password confirmation", with: "boomstick!3vilisd3ad"
+    attach_file :user_avatar, "#{Rails.root}/spec/support/images/avatar.jpg"
+    click_button "Sign up"
+    visit "/users/edit"
+
+    expect(page).to have_content("Edit User")
+    # expect(page).to have_xpath('//img[@src="/public/images/avatar.png"]')
+    # page.find('avatar')['src'].should have_content 'avatar.png'
+    expect(page).to have_css('.avatar')
```

Comment:
Others may disagree, but I prefer what you had on line 16.

Maybe try:
```
image_file = page.find('.avatar')['src']
expect(image_file).to include 'avatar.png'
```
or something along those lines. Put a `binding.pry` in there, and see what tags work!